static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -198,9 +210,13 @@ const canShow = async (): Promise<boolean> => {
     if (!canShowPreChecks({
         brazeSwitch,
         apiKey,
-        isDigiSubscriber: isDigitalSubscriber(),
+        userIsGuSupporter: shouldNotBeShownSupportMessaging(),
         pageConfig: config.get('page'),
     })) {
+        // Currently all active web canvases in Braze target existing supporters,
+        // subscribers or otherwise those with a Guardian product. We can use the
+        // (inverse) value of `shouldNotBeShownSupportMessaging` to identify these users,
```

Comment:
Minor (and I realize this is wrong in the DCR pr too) but I think we should remove `(inverse)` from this, since it's not accurate.

Reply:
Should have spotted that! Sorted now.